src/main/java/fr/inria/coming/codefeatures/RepairnatorFeatures.java

```diff
@@ -0,0 +1,208 @@
+package fr.inria.coming.codefeatures;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import add.main.Config;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+import add.entities.RepairPatterns;
+import add.features.detector.repairpatterns.RepairPatternDetector;
+
+import org.apache.log4j.Logger;
+import org.junit.Test;
+
+import fr.inria.coming.main.ComingProperties;
+import fr.inria.prophet4j.utility.CodeDiffer;
+import fr.inria.prophet4j.utility.Option;
+import fr.inria.prophet4j.utility.Option.FeatureOption;
+import fr.inria.prophet4j.utility.Structure.FeatureMatrix;
+import gumtree.spoon.AstComparator;
+import gumtree.spoon.diff.Diff;
+
+import ml.dmlc.xgboost4j.java.DMatrix;
+import ml.dmlc.xgboost4j.java.Booster;
+import ml.dmlc.xgboost4j.java.XGBoost;
+
+public class RepairnatorFeatures {
+
+	protected static Logger log = Logger.getLogger(Thread.currentThread().getName());
+    private final String UNKNOWN = "unknown";
+    private final String CORRECT = "correct";
+    private final String OVERFITTING = "overfitting";
+
+	
+	public String getLabel(File pairFolder) throws Exception {
+		
+		//Obtain feature in libsvm format
+		String features = extractFeatures(pairFolder);
+		if("".equals(features)) {
+			return UNKNOWN;
+		}
+		//Load test data in libsvm format
+		File tempFile = File.createTempFile("test", ".txt");
+		try (FileWriter sb = new FileWriter(tempFile)) {
+			sb.append(features);
+		}
+		//Load model
+		DMatrix dtest = new DMatrix(tempFile.getAbsolutePath());
+		String odsModel = this.getClass().getClassLoader().getResource("ODSTraining/ODSmodel.bin").getPath();  
```

Comment:
It would be nice if the can removed the hardcoded path to the trained mode. 
Can we pass it as:
1) as method parameter? 
2) as constructor parameter?
3) using Configuration properties mechanism?  (see an example of access to config properties [here](https://github.com/SpoonLabs/coming/blob/master/src/main/java/fr/inria/coming/codefeatures/FeatureAnalyzer.java#L96))?

WDYT?

Reply:
Sure, I will give an update in next commit. I failed to push another file which cause the build failure. Will update it as well.